components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -1,11 +1,14 @@
 import { Injectable } from '@angular/core';
```

Comment:
This file contains what is really all the front-end tweaks needed to use names instead of ids.